content/blog/set-up-load-balancer-with-metallb-in-hpe-greenlake-for-private-cloud-enterprise.md

```diff
@@ -0,0 +1,234 @@
+pod/speaker-zfl7s                1/1     Running   0          36s
+
+NAME                      TYPE        CLUSTER-IP       EXTERNAL-IP   PORT(S)   AGE
+service/webhook-service   ClusterIP   10.105.154.106   <none>        443/TCP   38s
+
+NAME                     DESIRED   CURRENT   READY   UP-TO-DATE   AVAILABLE   NODE SELECTOR            AGE
+daemonset.apps/speaker   6         6         4       6            4           kubernetes.io/os=linux   37s
+
+NAME                         READY   UP-TO-DATE   AVAILABLE   AGE
+deployment.apps/controller   0/1     1            0           38s
+
+NAME                                   DESIRED   CURRENT   READY   AGE
+replicaset.apps/controller-7967ffcf8   1         1         0       38s
+```
+
+#### 2 Define a range of IP addresses
+
+After all MetalLB components are deployed, you can start creating and allocating a range of IP addresses, which can be used by MetalLB to assign IP addresses to services. 
+
+The customer resource definition (CRD) *IPAddressPool* will be used for defining the range of IP addresses. After it’s deployed to the cluster, all the IP addresses will be allocated for MetalLB to use. 
+
+```markdown
+$ cat IPAddressPool.yaml 
+apiVersion: metallb.io/v1beta1
+kind: IPAddressPool
+metadata:
+  name: cfe-pool
+  namespace: metallb-system
+spec:
+  addresses:
+- 172.16.17.250-172.16.17.254
+
+$ kubectl apply -f IPAddressPool.yaml 
+ipaddresspool.metallb.io/cfe-pool created
+
+$ kubectl get IPAddressPool -n metallb-system
+NAME       AUTO ASSIGN   AVOID BUGGY IPS   ADDRESSES
+cfe-pool   true          false             ["172.16.17.250-172.16.17.254"]
+```
+
+The above command allocates the IP pool that has the IP range 172.16.17.250-172.16.17.254. The IP addresses in the *IPAddressPool* can be defined by *CIDR* and *IPV6* addresses as well.
+
+#### 3 Announce the service IP addresses
+
+Once the IP addresses are allocated, you must announce service IPs. The [MetalLB Configuration site](https://metallb.universe.tf/configuration/#announce-the-service-ips) shows a list of configuration approaches you can use to announce service IPs. The below example shows the details of using the *Layer 2* mode to configure service IP addresses. This approach does not need any protocol specific configuration, only IP addresses from the *IPAddressPool*.
+
+```markdown
+$ cat L2Advertisement.yaml 
+apiVersion: metallb.io/v1beta1
+kind: L2Advertisement
+metadata:
+  name: example
+  namespace: metallb-system
+spec:
+  ipAddressPools:
+  - cfe-pool
+
+$ kubectl apply -f L2Advertisement.yaml 
+l2advertisement.metallb.io/example created
+
+$ kubectl get L2Advertisement -n metallb-system
+NAME      IPADDRESSPOOLS   IPADDRESSPOOL SELECTORS   INTERFACES
+example   ["cfe-pool"]               
+```
+
+### D﻿eploy Nginx app as the service type *LoadBalancer*
+
+As a sample web application, the *Nginx* with the service type of *LoadBalancer* will be deployed to the K8s cluster using the following YAML manifest file:
```

Comment:
Maybe point out the fact that you have a custom index.html since you'll be showing it in the next step.